dataset.py

```diff
@@ -28,10 +28,15 @@ def load_dataset(data, data_dir, split, download):
                     transforms.ToTensor(),
                     normalize,
                 ]), download=download)
-        else:
+        elif split == 'test':
             return cifar_sets[data](root=data_dir, train=False, transform=transforms.Compose([
                     transforms.ToTensor(),
                     normalize,
                 ]), download=download)
+        else:
```

Comment:
Would it be better if we have `else if split == 'eval_train':` in case we have other types of split later?